lib/ecto_cursor_based_stream.ex

```diff
@@ -72,6 +73,7 @@ defmodule EctoCursorBasedStream do
   end
 
   @doc false
+  @spec call(Ecto.Repo.t(), Ecto.Queryable.t(), cursor_based_stream_opts) :: Enumerable.t()
```

Comment:
```suggestion
  @spec call(Ecto.Repo.t(), Ecto.Queryable.t(), opts :: Keyword.t()) :: Enumerable.t()
```

Reply:
mine is more correct, it provide the list of possible. Just need to use union type if you want more possible options

Reply:
oh sorry I didn't realize it was a type! my bad~